eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -4,6 +4,56 @@ parameters:
   TestMarkArgument: ''
   BuildTargetingString: 'azure-*'
   TestTimeoutInMinutes: 0
+  matrix:
```

Comment:
`Matrix` seems super ambivalent as a name. However, thinking about it I'm not certain how you can get much better. `AgentMatrix` maybe?

Reply:
Updated.